src/subdomains/generic/user/models/user-data/user-data.service.ts

```diff
@@ -410,8 +363,7 @@ export class UserDataService {
     const userDataList = await this.userDataRepo.findBy({ kycFileId: MoreThan(0) });
     const idList = [];
     for (const userData of userDataList) {
-      const customerInfo = await this.spiderApiService.getCustomerInfo(userData.id);
-      if (customerInfo && !customerInfo.contractReference) idList.push(userData.id);
+      idList.push(userData.id);
```

Comment:
define idList with map Method?
`return userDataList.map((u) => u.id)`

Reply:
The method `getAllUserDataWithEmptyFileId` seems to be obsolete. I would remove it.